stream/classes/cost_model/scheduler.py

```diff
@@ -234,7 +211,7 @@ def check_for_removal(
 def schedule_graph(
     G: DiGraph,
```

Comment:
Is it fine that here is `DiGraph` and not `Workload`?

Reply:
Both work, I guess I forgot to change this one to `Workload`. Note that the type hints are all but finished, and don't really matter that much for the moment (they are not actually checked).